autorest/adal/token_test.go

```diff
@@ -868,7 +862,7 @@ func expireToken(t *Token) *Token {
 
 func setTokenToExpireAt(t *Token, expireAt time.Time) *Token {
 	t.ExpiresIn = "3600"
-	t.ExpiresOn = strconv.Itoa(int(expireAt.Sub(date.UnixEpoch()).Seconds()))
+	t.ExpiresOn = json.Number(strconv.Itoa(int(expireAt.Sub(date.UnixEpoch()).Seconds())))
```

Comment:
I'd change this to
``` Go
t.ExpiresOn = json.Number(fmt.Sprintf(int64(expireAt.Sub(date.UnixEpoch() / time.Seconds))))
```

While it's not any more readable, it has two advantages IMHO:
1. Not calling `.Seconds()` prevents the conversion from an `int` type to a `float` type.
1. Using `fmt.Sprintf()` instead of `strconv.Itoa()` prevents us from casting down from 64 bits to 32 bits.

Reply:
Errr, I see this is just a test file so don't see this comment as too blocking or anything.

Reply:
I tweaked this a bit.